storybookjs/storybook
# Docs: Adds set up wizard information

With this small pull request, the setup page was updated to feature some information about the setup wizard that's enabled when a user installs Storybook.

What was done:
- Updated the wording on the install page and made it conditionally render it.
- Updated the required assets
## How to test

1. Follow the steps in the [contributing instructions](https://storybook.js.org/docs/react/contribute/new-snippets#preview-your-work) for this branch, `docs_add_tour_info`

2. Open the documentation, and for React, you'll see:

<img width="1440" alt="install-storybook-wizard" src="https://github.com/storybookjs/storybook/assets/22988955/2f479307-966c-4ffe-b504-87af9838a929">

3. Open the documentation for any other framework and (e.g., Vue) you'll see:

<img width="1440" alt="install-storybook-non-setup-wizard" src="https://github.com/storybookjs/storybook/assets/22988955/0a526d7c-fd13-4ef2-9af7-c5848d981dee">


@valentinpalkovic and @cdedreuille, I've requested both of you as reviewers. When you have a moment, take a look and let me know of any feedback. Thanks in advance
